docs/manual/optional-checker.tex

```diff
@@ -56,11 +56,26 @@
 
 \sectionAndLabel{How to run the Optional Checker}{optional-run-checker}
 
+The standard way to run the Optional Checker is one of these command lines:
+
 \begin{Verbatim}
 javac -processor optional MyFile.java ...
 javac -processor org.checkerframework.checker.optional.OptionalChecker MyFile.java ...
 \end{Verbatim}
 
+% TODO: This is gross.  Figure out a way to control this from the Optional Checker.
```

Comment:
Change this into mentioning the existence of the OptionalWithoutNonEmptyChecker, and mention the downside.